code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -161,9 +164,32 @@ def get_default_config():
                         ORCHESTRATION_STRATEGY=env_helper.ORCHESTRATION_STRATEGY
                     )
                 )
+                if env_helper.USE_ADVANCED_IMAGE_PROCESSING:
+                    ConfigHelper._append_advanced_image_processing_processors()
```

Comment:
do we need the `_processing_` :-) 

```suggestion
                    ConfigHelper._append_advanced_image_processors()
```

Reply:
Good point! I'll remove!